docs/docs/recipes/deploying-your-site.md

```diff
@@ -1,132 +1,132 @@
 
 ```shell
 gatsby build
 ```
 
-3. To include a site path other than `/` (such as `/site-name/`), set a path prefix by adding the following to your `gatsby-config.js` and replacing `yourpathprefix` with your desired path prefix:
+3. Aby zawrzeć na stonie inną ścieżkę niż `/` (jak np. `/tytul-strony/`), ustaw przedrostek dodając poniższy kod do swojego `gatsby-config.js` i zamień ścieżkę `yourpathprefix` na swoją własną:
 
 ```js:title=gatsby-config.js
 module.exports = {
   pathPrefix: `/yourpathprefix`,
 }
 ```
 
-There are a few reasons to do this -- for instance, hosting a blog built with Gatsby on a domain with another site not built on Gatsby. The main site would direct to `example.com`, and the Gatsby site with a path prefix could live at `example.com/blog`.
+Istnieje kilka powodów by użyć -- ,dla przykładu hosting bloga zbudowanego za pomocą Gatsby na domenie z inną stroną, która nie jest zbudowana w Gatsby. Główna strona mogłaby przekierowywać do `przyklad.com`, a strona wykorzystująca Gatsby byłaby dostępna pod adresem `przyklad.com/blog`.
 
-4. With a path prefix set in `gatsby-config.js`, run `gatsby build` with the `--prefix-paths` flag to automatically add the prefix to the beginning of all Gatsby site URLs and `<Link>` tags.
+4. Jeśli ustawiłeś przedrostek w `gatsby-config.js`, uruchom `gatsby build` z flagą `--prefix-paths` aby automatycznie dodać przedrostek do początku adresu URL każdej ze stron oraz do tagów `<Link>`.
 
 ```shell
 gatsby build --prefix-paths
 ```
 
-5. Make sure that your site looks the same when running `gatsby build` as with `gatsby develop`. By running `gatsby serve` when you build your site, you can test out (and debug if necessary) the finished product before deploying it live.
+5. Upewnij się, że twoja strona wygląda tak samo używając komendy `gatsby build` jak i `gatsby develop`. Kiedy strona została zbudowana uruchom `gatsby serve`, możesz teraz przetestować jej finalny efekt(i jeśli to potrzebne - poprawić) tuż przed upublicznieniem.
 
 ```shell
 gatsby build && gatsby serve
 ```
 
-### Additional resources
+### Dodatkowe źródła
 
-- Walk through building and deploying an example site in [tutorial part one](/tutorial/part-one/#deploying-a-gatsby-site)
-- Learn about [performance optimization](/docs/performance/)
-- Read about [other deployment related topics](/docs/preparing-for-deployment/)
-- Check out the [deployment docs](/docs/deploying-and-hosting/) for specific hosting platforms and how to deploy to them
+- Przejdź przez samouczek dotyczący budowania i wdrażania przykładowej strony w [części drugiej](/tutorial/part-one/#deploying-a-gatsby-site)
+- Dowiedz się więcej o [optymalizacji wydajności](/docs/performance/)
+- Przeczytaj więcej o [innych możliwościach wdrażania strony](/docs/preparing-for-deployment/)
+- Sprawdź [dokumenty wdrożeniowe](/docs/deploying-and-hosting/) aby odnaleźć więcej informacji na temat konkretnych platform hostingowych oraz jak wykorzystując je - wdrożyć stronę.
```

Comment:
[dokumenty wdrożeniowe](/docs/deploying-and-hosting/) aby odnaleźć więcej informacji
change to
[dokumenty wdrożeniowe](/docs/deploying-and-hosting/), aby odnaleźć więcej informacji
// missing comma